apps/dapp-console-api/src/routes/faucet/FaucetRoute.ts

```diff
@@ -517,4 +548,14 @@ export class FaucetRoute extends Route {
       return 'GENERIC_ERROR'
     }
   }
+
+  private readonly updateFaucetClaimRate = async (ip?: string) => {
```

Comment:
I dont think we need to be checking the growthbook store many times here. It can be contractual that it is the callers responsibility to ensure this feature prior to calling this